addOns/pscanrules/src/test/java/org/zaproxy/zap/extension/pscanrules/InformationDisclosureSuspiciousCommentsScanRuleUnitTest.java

```diff
@@ -282,6 +333,47 @@ void shouldAlertOnSuspiciousCommentProvidedByCustomPayload()
         assertEquals(1, alertsRaised.size());
     }
 
+    @ParameterizedTest
+    @FieldSource(
+            "org.zaproxy.zap.extension.pscanrules.InformationDisclosureSuspiciousCommentsScanRule#DEFAULT_PAYLOADS")
+    void shouldAlertOnSuspiciousCommentInJavascriptSingleLineCommentProvidedByDefaultCustomPayloads(
```

Comment:
IMO does not worth test with all payloads, one is enough to highlight the case.

Reply:
Worth updating the test names too.

Reply:
I mean the part `ProvidedByDefaultCustomPayloads`.

Reply:
Tweaked that too now